docs/content/tutorial/ingress-tutorial.md

```diff
@@ -0,0 +1,139 @@
+---
+title: "Exposing functions with Ingress"
+draft: false
+weight: 43
+---
+
+This tutorial will walk you through exposing a function using a ingress controller. We will make the function available on a FQDN using Fission's route and ingress controller setup in a cloud environment.
+
+## Setup & pre-requisites
+
+You will need a Kubernetes cluster with Fission installed (Please check [installation page](../../installation/installation/) for details). This tutorial uses a cloud load balancer and if you are trying on Minikube, you might want [to take a look at details here](https://github.com/kubernetes/minikube/issues/496)
+
+Later parts of this tutorial use a FQDN to reach the function. If you plan to go along in this section, you will need a domain name setup and access to modify the NS records & create A record in the zone of the domain name you have. The tutorial uses Google cloud to walk through the tutorial but you can use any cloud you prefer to. Also the changes in name server can take upto 24-48 hours so you may want to use an already created domain name.
+
+### Setup an Ingress Controller
+
+First thing we will need is an ingress controller and we will use Nginx ingress controller in this tutorial. Based on your setup you can choose [one of the ways to install Nginx ingress controller from here](https://kubernetes.github.io/ingress-nginx/deploy/). This setup should work with other ingress controllers also but has not been tested.
```

Comment:
one of the ways to install Nginx ingress controller from here -> one of multiple ways to install Nginx ingress controller 

Try to avoid using "click here" or "from here" in links.